spec/models/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
```

Comment:
- [ ] Make sure your app has integration tests for all the most important components of the RoR application 

- Spec file include tests that ensures correct view template is rendered. Example : `expect(response).to render_template(:show)`
- Spec file include tests that ensures the response body includes the correct placeholder text. Example : `expect(response.body).to include("Welcome to user index page")`